eclass/toolchain.eclass

```diff
@@ -277,6 +277,7 @@ if [[ ${PN} != kgcc64 && ${PN} != gcc-* ]] ; then
 	tc_version_is_at_least 10 && IUSE+=" zstd" TC_FEATURES+=( zstd )
 	tc_version_is_at_least 11 && IUSE+=" valgrind" TC_FEATURES+=( valgrind )
 	tc_version_is_at_least 11 && IUSE+=" custom-cflags"
+	tc_version_is_at_least 11 && IUSE+=" ieee-long-double"
```

Comment:
I suspect we don't want to do this for 11, only 12+, to narrow the things we're actually testing with.